banzai/utils/date_utils.py

```diff
@@ -133,10 +133,14 @@ def valid_date(s):
         raise argparse.ArgumentTypeError(msg)
 
 
-def get_min_and_max_dates(timezone, dayobs, return_string=False):
+def get_min_and_max_dates(timezone, dayobs, return_string=False, single_night=False):
+    if single_night:
+        days_minus, days_plus = 0.5, 0.5
```

Comment:
This feels more confusing than it needs to be. Why not just do the min_date and max_date calculations inside the if statement?

Reply:
Good point, I've changed it!